src/Files.App/ViewModels/UserControls/Previews/ShellPreviewViewModel.cs

```diff
@@ -198,8 +206,9 @@ public void UnloadPreview()
 		{
 			if (hwnd != HWND.NULL)
 				DestroyWindow(hwnd);
-			if (outputLink is not null)
-				outputLink.Dispose();
+
+			outputLink?.Dispose();
```

Comment:
Nit: Should probably `outputLink = null` afterwards to avoid a dangling pointer.

Reply:
Thanks!